plugins/modify-secret.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: modify-secret
+spec:
+  version: v0.0.3
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-modify-secret/releases/download/v0.0.3/darwin-amd64-v0.0.3.tar.gz
+    sha256: 809327469859fa665a6944e73975e0d0542c997ab048b091afc725b8e6413438
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-modify-secret
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-modify-secret/releases/download/v0.0.3/linux-amd64-v0.0.3.tar.gz
+    sha256: 9450d2c544859c18ad881e805d9fd9c3ec5bad5f82f6d6fba187f2e29706f672
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-modify-secret
+  shortDescription: kubectl-modify-secrets allows user to directly modify the secret without worrying about base64 encoding/decoding
```

Comment:
The short description gets truncated after ~50 characters when running `krew search`. It improves the discoverability of your plugin if you come up with a more succinct description.

Reply:
done. changed it to "modify secret with implicit base64 translations"